plugins/kotlin/idea/tests/testData/quickfix/replaceWithSafeCall/Test1.kt

```diff
@@ -0,0 +1,12 @@
+// "Replace with safe (?.) call" "true"
```

Comment:
Please use proper formatting in tests (and no redundant empty lines) and meaningful file names.